### ISSUE TITLE
`nexus-prisma` is handed over to the community for more active maintenance & future development 🚀

### ISSUE THREAD

**Issue by nikolasburk** [graphql-nexus/nexus-prisma#279]
Hey everyone 👋 we are using this issue to share some news with you today!

### Handing over `nexus-prisma` to the community 🎉

As you might have noticed, the `nexus-prisma` repo has been transferred from the `prisma` to the `graphql-nexus` org on GitHub. 

This is because we (Prisma) have turned the maintenance and development of the `nexus-prisma` repo into a community project with @rostislav-simonik being the core contributor going forward. We are extremely grateful to Rostislav for his openness to take over the `nexus-prisma` package and excited to see where the community will take it!

At Prisma we haven't been able to give `nexus-prisma` the attention it deserves in the last couple of months. Its development often lagged behind Prisma releases, so that `nexus-prisma` users often were stuck on older versions of Prisma.

With the community handover, we're confident that the `nexus-prisma` releases can keep up with new features being added to the Prisma ORM!

### FAQ

**What does this mean for me?**

Nothing really, you'll be able to continue `nexus-prisma` as before. In case you're interested in contributing to the `nexus-prisma` repo, feel free to reach out to Rostislav directly.

**What if I want to migrate off of `nexus-prisma`?**

The TypeScript ecosystem comes with a few of alternatives to `nexus-prisma` that offer an integration with Prisma:
- [Pothos](https://pothos-graphql.dev/docs/plugins/prisma) ([Tutorial](https://www.prisma.io/blog/e2e-type-safety-graphql-react-3-fbV2ZVIGWg) | [Prisma Day 22 talk](https://www.youtube.com/watch?v=LqKPfMmxFxw&ab_channel=Prisma))
- [TypeGraphQL](https://prisma.typegraphql.com/) ([Tutorial](https://dev.to/prisma/prototyping-a-crud-api-with-typegraphql-and-prisma-for-your-database-424c) | [Prisma Day 21 talk](https://www.youtube.com/watch?v=v6cNeHCfSHs&ab_channel=Prisma))

**What work will be done in the package next?**

The main focus of the development in the next weeks will be to make sure `nexus-prisma` is usable with the latest Prisma versions. There will also be some triaging and prioritization of existing issues. Feel free to share your thoughts on a future direction for `nexus-prisma` and the things you'd like to see!

**Comment by rostislav-simonik** [comment]
Thank you @nikolasburk  for the introduction. I’d like to continue on maintenance on nexus-prisma for two specific reasons.

- To support existing users of nexus nexus-prisma, who don’t have the capacity to migrate to very promising alternatives such as Pothos, and TypeGraphQL
- To keep pace for nexus with prisma updates. This is still important for nexus users, who prefer the nexus concept of types generation.

As @nikolasburk has mentioned I’d like to prioritize first regular cuts of releases, and then look into current issues.

**Comment by Frosty21** [comment]
Congrats @rostislav-simonik if there is anything I can do to help out let me know.

- I think first thing we should be addressing is chore(deps) in hand at the moment. 
- Then look into improving the documentation (I'll look into helping here)

**Comment by rostislav-simonik** [comment]
@Frosty21 yes, sure I'm gonna look this weekend at how we could enable automatic releases for minor, patch chore(deps).

**Comment by fullStackDataSolutions** [comment]
This is great news. I think Nexus is a great framework and its too bad support ended. Nexus-Prisma has saved countless hours and is an amazing bit of code!

**Comment by iddan** [comment]
@rostislav-simonik I'd love to join the repository contributors. Been helping the Prisma team to progress a lot with this package last summer.

**Comment by rostislav-simonik** [comment]
@iddan that's good news, I wrote a suggestion for roadmap #306, feel free to review and suggest anything you wish. If you are ok to deal chore(deps) first you can start on that,  I'll give you access. I'm interested in PR reviews, but if everything passes, and I won't respond within 24 hours, you can self-approve, and merge them as well.

**Comment by medet-mattr** [comment]
hi, any updates on nexus-prisms?

**Comment by rostislav-simonik** [comment]
@medet-mattr I made some progress two weeks ago but got temporarily busy with something else, I'm planning to make some progress next week.
